Client/Frontend/Browser/BrowserTrayAnimators.swift

```diff
@@ -304,20 +304,25 @@ private func createTransitionCellFromTab(_ tab: Tab?, withFrame frame: CGRect) -
     cell.screenshotView.image = tab?.screenshot
     cell.titleText.text = tab?.displayTitle
 
-    if let tab = tab, tab.isPrivate {
+    let tabType = TabType.of(tab)
+    
+    switch tabType {
```

Comment:
I used a switch statement so that a compiler error is thrown if/when a new 'case' is added, making sure that we think about the implementation for each case